polygeist/include/mlir/Dialect/Polygeist/IR/PolygeistTypes.td

```diff
@@ -32,8 +32,14 @@ def Polygeist_StructType :
     Type used to represent a collection of data members together in memory.
   }];
   let parameters = (ins ArrayRefParameter<"mlir::Type">:$body,
-                        OptionalParameter<"std::optional<StringAttr>">:$name);
-  let assemblyFormat = "`<` ($name^`,` )? `(` $body `)` `>`";
+                        OptionalParameter<"std::optional<StringAttr>">:$name,
+                        DefaultValuedParameter<"bool", "false">:$isPacked);
+  let builders = [
+    TypeBuilder<(ins "llvm::ArrayRef<mlir::Type>":$body),[{
+      return $_get($_ctxt, body, std::nullopt, false);
+    }]>
+  ];
```

Comment:
Are we using this constructor anywhere?

Reply:
not right now, it will used in CodeGenType.cc, do you prefer me to defer this builder change?

Reply:
Nah, this is fine :+1: